webapp/channels/src/components/emoji_picker/components/emoji_picker_skin.tsx

```diff
@@ -3,27 +3,74 @@
 
 import classNames from 'classnames';
 import React from 'react';
-import {FormattedMessage, injectIntl} from 'react-intl';
-import type {IntlShape} from 'react-intl';
+import {FormattedMessage, defineMessage, injectIntl} from 'react-intl';
+import type {IntlShape, MessageDescriptor} from 'react-intl';
 import {CSSTransition} from 'react-transition-group';
 
 import {CloseIcon} from '@mattermost/compass-icons/components';
+import type {SystemEmoji} from '@mattermost/types/emojis';
 
 import WithTooltip from 'components/with_tooltip';
 
 import imgTrans from 'images/img_trans.gif';
 import * as Emoji from 'utils/emoji';
 
-const skinsList = [
-    ['raised_hand_with_fingers_splayed_dark_skin_tone', '1F3FF'],
-    ['raised_hand_with_fingers_splayed_medium_dark_skin_tone', '1F3FE'],
-    ['raised_hand_with_fingers_splayed_medium_skin_tone', '1F3FD'],
-    ['raised_hand_with_fingers_splayed_medium_light_skin_tone', '1F3FC'],
-    ['raised_hand_with_fingers_splayed_light_skin_tone', '1F3FB'],
-    ['raised_hand_with_fingers_splayed', 'default'],
-];
+interface SkinTone {
+    emoji: SystemEmoji;
+    label: MessageDescriptor;
+    value: string;
+}
 
-const skinToneEmojis = new Map(skinsList.map((pair) => [pair[1], Emoji.Emojis[Emoji.EmojiIndicesByAlias.get(pair[0])!]]));
+const skinTones = [
```

Comment:
Instead of deriving these from an array of tuples, I just wrote it out for simplicity's sake which is similar to many of the other changes in this PR. This wasn't strictly necessary, but it felt nicer to do that than trying to add `defineMessage` to the previous `skinsList` without upsetting TS